pkg/accounting/rollup/rollup.go

```diff
@@ -66,21 +66,25 @@ func (r *Service) Rollup(ctx context.Context) error {
 	if err != nil {
 		return Error.Wrap(err)
 	}
-	if len(rollupStats) == 0 {
-		r.logger.Info("RollupStats is empty after RollupStorage")
-	}
+
 	err = r.RollupBW(ctx, lastRollup, rollupStats)
 	if err != nil {
 		return Error.Wrap(err)
 	}
+
+	//remove the latest day (which we cannot know is complete), then push to DB
+	latestTally = time.Date(latestTally.Year(), latestTally.Month(), latestTally.Day(), 0, 0, 0, 0, latestTally.Location())
+	delete(rollupStats, latestTally)
 	if len(rollupStats) == 0 {
-		r.logger.Info("RollupStats is empty after RollupBW")
+		r.logger.Info("RollupStats is empty")
 		return nil
 	}
+
 	err = r.db.SaveRollup(ctx, latestTally, rollupStats)
 	if err != nil {
 		return Error.Wrap(err)
 	}
+
 	// Delete already rolled up tallies
 	err = r.db.DeleteRawBefore(ctx, latestTally)
```

Comment:
let's wrap this in a conditional statement that checks the config

Reply:
I implemented this, but need to figure out how to set flags in testplanet because it's being set to false and testrollup is failing as a result because we assert that raws were deleted